server/controllers/index.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../models');
```

Comment:
Out of curiosity why did you separate these functions out from your routes? I'm not sure what I think about it. If you were re-using some of these functions I would applaud you keeping things dry, and I'm going to keep this idea in mind. Just wondering if you had a specific purpose for separating them.

Reply:
I went back and forth over the file structure, and ultimately went with separating out [routes, models, and controllers](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes). I realize it may be over-kill for a service of this size, but I wanted to practice building the structure used for scale. I also looked back to our databases sprint for reference.